extremeflash/ws.py

```diff
@@ -215,7 +219,10 @@ def boot_via_tftp(
         write_to_serial(ser, b"bootm\n", sleep=0.1)
 
     logging.info("Starting TFTP Boot.")
+    wait_for_ramboot(ser)
```

Comment:
To be consistent, this function should probably be moved to start_tftp_boot_via_serial, like the "wait_for_brlan"